jupyterlab_polus_render/src/plugin.ts

```diff
@@ -71,13 +80,178 @@ function activateWidgetExtension(
           });
         });
       });
+    }
+
+    protected showUnsupportedFileTypePopup() {
+      showDialog({
+        title: 'Unsupported File Type',
+        body: `The file format is not supported.`,
+        buttons: [Dialog.okButton({ label: 'OK' })]
+      });
+    }
+
+    /**
+     * Handle the lm-dragenter event for the widget.
+     */
+    protected handleDragEnter(event: Drag.Event): void {
+      event.preventDefault();
+      event.stopPropagation();
+      // Access CSS class: dragover
+      this.dropzoneElement.classList.add('dragover');
+      this.dropzoneElement.textContent = 'Drop files here'; // Show text when entering 
+      console.log("Drag enter event:", event);
```

Comment:
Please, remove these log messages, they create so many unnecessary lines in the browser console